chain/chain/src/store.rs

```diff
@@ -1956,6 +1957,50 @@ impl<'a> ChainStoreUpdate<'a> {
         self.chunk_tail = Some(height);
     }
 
+    pub fn clear_chunk_data_and_headers_at_height(
```

Comment:
```suggestion
   fn clear_chunk_data_and_headers_at_height(
```
because it is not called from outside, and it looks like impl detail of undoing block